src/peer_map.rs

```diff
@@ -129,6 +129,12 @@ impl PeerMap {
     pub fn has<N: AsRef<XorName>>(&self, name: N) -> bool {
         self.forward.contains_key(name.as_ref())
     }
+
+    // If we don't have any information about this peer when this query is made then in certain
+    // context it could mean it's likely a client
+    pub fn is_peer_known_to_routing(&self, peer_addr: &SocketAddr) -> bool {
```

Comment:
I'd call this `is_node` because I think it better describes the purpose of the method (whether the connection is from client or node). Also it's shorter :D